### PR TITLE
Implement Route and RouterInterface APIs

### PR DESCRIPTION
**Pull request by DanG100**

### CODE REVIEW

File: dataplane/standalone/saiserver/routing.go [hunk 1]
```diff
@@ -175,6 +175,62 @@ func newRoute(mgr *attrmgr.AttrMgr, dataplane routingDataplaneAPI, s *grpc.Serve
...
+				},
+				VrfId: req.GetEntry().GetVrId(),
+			},
+		},
+	}
+
+	// TODO(dgrau): Implement CPU actions.
+	switch req.GetPacketAction() {
+	case saipb.PacketAction_PACKET_ACTION_DROP:
+		fallthrough
```
Comment (alexmasi) [inline dataplane/standalone/saiserver/routing.go:197]: I think using fallthrough can be a bit confusing here, why not group the cases?

```
case saipb.PacketAction_PACKET_ACTION_DROP, 
     saipb.PacketAction_PACKET_ACTION_TRAP, 
     saipb.PacketAction_PACKET_ACTION_DENY:
     rReq.Route.Action = dpb.PacketAction_PACKET_ACTION_DROP
```

File: dataplane/standalone/saiserver/routing.go [hunk 2]
```diff
@@ -189,3 +245,29 @@ func newRouterInterface(mgr *attrmgr.AttrMgr, dataplane routingDataplaneAPI, s *
...
+		VrfId:   uint32(req.GetVirtualRouterId()),
+		Mtu:     uint64(req.GetMtu()),
+		PortIds: []string{fmt.Sprint(req.GetPortId())},
+		Mac:     req.GetSrcMacAddress(),
+	}
+	switch req.GetType() {
+	case saipb.RouterInterfaceType_ROUTER_INTERFACE_TYPE_PORT:
+		iReq.Type = dpb.InterfaceType_INTERFACE_TYPE_PORT
+
+	case saipb.RouterInterfaceType_ROUTER_INTERFACE_TYPE_LOOPBACK: // TODO: Support loopback interfaces
```
Comment (alexmasi) [inline dataplane/standalone/saiserver/routing.go:263]: should this log a warning or something to indicate that its not yet fully supported? or maybe even return an error

**Review by alexmasi — APPROVED** [review]

File: dataplane/standalone/saiserver/routing.go (hunk 1 above)
Comment (DanG100) [reply dataplane/standalone/saiserver/routing.go]: done

File: dataplane/standalone/saiserver/routing.go (hunk 2 above)
Comment (DanG100) [reply dataplane/standalone/saiserver/routing.go:263]: done

**Pull request merged**